Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
I gave a second language that compiles to similar byte code and has the same public-only interface definitions. Imo there should be way more compiler errors in Java, but because of how it evolved and backwards compatibility, most errors are shown as warnings.

Do you guys run Android lint?